app/models/user.rb

```diff
@@ -8,7 +8,7 @@ class User < ApplicationRecord
 
   validates_presence_of :daac, if: Proc.new { |u| u.role.eql?("daac_curator") }
 
-  ROLES = %w[admin arc_curator daac_curator].freeze
+  ROLES = %w[admin arc_curator mdq_curator daac_curator].freeze
```

Comment:
earthdata_curator maybe, but I don't like mdq

Reply:
The ACL in CMR is called MDQ curator, so changing it to something else would only complicate the matter.

Reply:
we can ask to change the ACL, and therefore change it here if we want. like we might want to change the ACL name for subscriptions.